index.html

```diff
@@ -31,7 +50,128 @@
+        <i class="closeBtn hide"></i>
+        <i class="fa-sharp fa-solid fa-bars hamburger"></i>
+      </nav>
+    </header>
+    <section class="hero">
+      <div class="mainHero">
+        <div class="contentHero">
+          <h5 class="introHero">"Hello! World of Music"</h5>
+          <h1 class="titleHero">GLOBAL MUSIC CONCERT SAGA 2023</h1>
+        </div>
+        <div class="descHero">
+          <p class="desc">
+            A joyful celebration believing in the value of openness and sharing,
+            creating a positive change with music for the people from over 80
+            countries is taking place in April, in South Korea.
+          </p>
+        </div>
+        <div class="venueHero">
+          <h5 class="date">2023.04.24 (THU) ~ 30 (FRI)</h5>
+          <p class="venue">
+            @Open Air Theatre of Seoul, South Korea, Music Center and more...
+          </p>
+        </div>
+      </div>
+    </section>
+    <section class="mainProgram">
+      <div class="mainProgramTitle">
+        <div class="headingMainProgram">
+          <h4 class="titleHeading">Main Program</h4>
+          <div class="borderBottom"></div>
+        </div>
+        <div class="categories">
+          <div class="category">
+            <img class="icon" src="images/program_icon_01.svg" alt="" />
+            <h5 class="categoryTitle">Lecture</h5>
+            <p class="categoryDesc">
+              Listen to the speakers from various countries about the messages
+              of sharing and opening.
+            </p>
+          </div>
+          <div class="category">
+            <img class="icon" src="images/program_icon_02.svg" alt="" />
+            <h5 class="categoryTitle">CC Exhibition</h5>
+            <p class="categoryDesc">
+              Appreciate various creations applying CC license of artists,
+              organized from Art Center Nabi
+            </p>
+          </div>
+          <div class="category">
+            <img class="icon" src="images/program_icon_03.svg" alt="" />
+            <h5 class="categoryTitle">Forum</h5>
+            <p class="categoryDesc">
+              Have the time to share your thoughts and opinions with experts for
+              each topic.
+            </p>
+          </div>
+          <div class="category">
+            <img class="icon" src="images/program_icon_04.svg" alt="" />
+            <h5 class="categoryTitle">Workshop</h5>
+            <p class="categoryDesc">
+              Try creating your own work using open source license rather than
+              just watching at it.
+            </p>
+          </div>
+          <div class="category">
+            <img class="icon" src="images/program_icon_05.svg" alt="" />
+            <h5 class="categoryTitle">CC Ignite</h5>
+            <p class="categoryDesc">
+              get opportunities to network with CC affiliates around the world,
+              also after the summit.
+            </p>
+          </div>
+        </div>
```

Comment:
- [ Required ] Please consider adding an `alt` tag for all the images on your homepage, an `alt` tag makes an image accessible to someone who is using a web accessibility tool like a screen reader or in cases where the image can not load the `alt` text is always displayed.